viewing_party/party.py

```diff
@@ -1,23 +1,132 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating: 
+        return { "title": title, "genre": genre, "rating": rating}
+    else:
+        return None

```

Comment:
This else is not needed! Functions automatically return `None` if there is no return statement. So if the function leaves the if statement and does not return a movie dictionary, it will automatically return `None`.